generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -178,6 +178,8 @@ class DividedClockRocketConfig extends Config(
   new chipyard.config.WithTileFrequency(200.0) ++
   new freechips.rocketchip.subsystem.WithRationalRocketTiles ++   // Add rational crossings between RocketTile and uncore
   new freechips.rocketchip.subsystem.WithNBigCores(1) ++
+  new chipyard.WithMemoryBusFrequency(50 * 1000 * 1000) ++
```

Comment:
The disconnect between this and the tile frequency is annoying. IMO this API should also be in MHz.
Also to clarify this config has tiles: 200, sbus+l2: 100, and mbus: 50?

Reply:
Yeah, it is. The dtsFrequency argument is a `BigInt` in units in Hz, but frequencies used by diplomatic clocks are `Double`s and are in units of MHz. I think my preference would be to rally about the latter.

Reply:
And yes, those are the correct frequencies.

Reply:
My preference is also for the latter